extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -90,23 +91,28 @@ iframe.swg-dialog {
   font-weight: 400 !important;
 }
 
-/* TODO(#38112): confirm with UX about if there's better animation to be used. */
+[dir="rtl"] .i-amphtml-story-subscriptions-dialog-banner-button {
+  transform: translate(0.2em) !important;
+}
+
 .i-amphtml-story-subscriptions-dialog-banner-button.i-amphtml-story-subscriptions-dialog-banner-button-visible {
   opacity: 1 !important;
+  transform: translate(0) !important;
 }
 
 .i-amphtml-story-subscriptions-dialog-banner-button:after {
   content: '' !important;
   width: 0.93em !important;
   height: 0.93em !important;
-  margin: 0 0.6em !important;
+  margin: 0 0.5em !important;
+  transform: translate(0.1em) !important;
   background-size: contain !important;
   background-repeat: no-repeat !important;
   background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" stroke="black" fill="none" stroke-width="2px" stroke-linecap="round" stroke-linejoin="round"><polyline points="1.75,1.5 6,6 1.75,10.6"/></svg>') !important;
 }
 
 [dir="rtl"] .i-amphtml-story-subscriptions-dialog-banner-button:after {
-  transform: scale(-1, 1) !important;
+  transform: scale(-1, 1) translate(0) !important;
```

Comment:
This wouldn't be 0, it would be like you did for the rtl block, it would be the reversed animation direction.

You need a new CSS rule to target the `after` when the button is active.

```
.i-amphtml-story-subscriptions-dialog-banner-button:after.i-amphtml-story-subscriptions-dialog-banner-button-visible {
  transform: translate(0) !important;
}
```

Reply:
Ah nice catch. Ok now `scale(-1, 1) translate(0.1em)` is still needed for RTL block. And the properties for the visible button would be the same, e.g.  `transform: translate(0) !important;`.